### PR TITLE
add separate leex files for rendering

### PR DESCRIPTION
**Pull request by jprincipe**

### CODE REVIEW

File: lib/game_of_life_web/live/universe_live.ex [hunk 1]
```diff
@@ -1,33 +1,19 @@
 defmodule GameOfLifeWeb.UniverseLive do
   use Phoenix.LiveView
-  use Phoenix.HTML, only: raw/1
   alias GameOfLife.Universe
 
   def render(assigns) do
-    ~L"""
```
Comment (BGRicker) [inline lib/game_of_life_web/live/universe_live.ex:7]: this is making much more sense to me now!

File: lib/game_of_life_web/views/cell_view.ex [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  use GameOfLifeWeb, :view
+
+  def render_cell(%{alive: alive}) do
+    case alive do
+      false -> content_tag(:div, nil, class: "cell dead")
+      true -> content_tag(:div, nil, class: "cell alive")
+      _ -> content_tag(:div, "-", class: "cell")
+    end
+  end
+end
```
Comment (kevlarr) [inline lib/game_of_life_web/views/cell_view.ex:7]: This is sooo nice

File: lib/game_of_life_web/views/universe_view.ex [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  def render_cells(cells, {width, height}) do
+    Enum.map(0..(height - 1), fn y ->
+      content_tag(:div, class: "cell-row") do
+        Enum.map(0..(width - 1), fn x ->
+          GameOfLifeWeb.CellView.render_cell(Map.get(cells, {x, y}))
+        end)
+      end
+    end)
+  end
+end
```
Comment (kevlarr) [inline lib/game_of_life_web/views/universe_view.ex:13]: This is awesome, I don't think I ever saw `content_tag` like this.. I love that it takes a block, too

File: lib/game_of_life_web/live/universe_live.ex [hunk 4]
```diff
@@ -1,33 +1,19 @@
...
 
-    socket = assign(socket,
-      universe: rand_bytes(),
-      dimensions: {8, 8}
-    )
+    socket =
+      assign(socket,
+        universe: rand_bytes(),
+        dimensions: {8, 8}
+      )
```
Comment (kevlarr) [inline lib/game_of_life_web/live/universe_live.ex:16]: Ah right, I forgot about style conventions on this one

**Review by kevlarr — APPROVED** [review]

File: lib/game_of_life_web/views/cell_view.ex [hunk 5]
```diff
@@ -0,0 +1,11 @@
+defmodule GameOfLifeWeb.CellView do
+  use GameOfLifeWeb, :view
+
+  def render_cell(%{alive: alive}) do
+    case alive do
+      false -> content_tag(:div, nil, class: "cell dead")
```
Comment (BGRicker) [inline lib/game_of_life_web/views/cell_view.ex]: this is awesome, love how it's setting up the HTML in this case statement

**Comment by kevlarr** [comment]
This is great!! I love seeing how to support better Phoenix conventions

**Review by BGRicker — APPROVED** [review]

**Pull request merged**